docker-compose.yml

```diff
@@ -36,9 +54,16 @@ services:
     depends_on:
      - config-server
      - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseCGroupMemoryLimitForHeap", "-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
+     - database-service
+    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s", "-wait=tcp://database-service:3306", "-timeout=60s", "--","java", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseCGroupMemoryLimitForHeap", "-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```

Comment:
tohle se netyka domaci ulohy, ale zmen obecne - tady je naznacena zmena, ale vypada to, ze jsou radky shodne a ze zmena s ukolem nesouvisi. reviewer ale vidi zmenu a musi se na to zamerit.

Reply:
Radky nejsou shodne, pridal jsem tam zavislost skrz ./dockerize plugin, aby sluzba cekala i na databazi a ne jen na discovery-service

Reply:
uz to vidim!